modules/nats/nats.go

```diff
@@ -33,7 +33,7 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 		Image:        img,
 		ExposedPorts: []string{defaultClientPort, defaultRoutingPort, defaultMonitoringPort},
 		Cmd:          []string{"-DV", "-js"},
-		WaitingFor:   wait.ForLog("Listening for client connections on 0.0.0.0:4222"),
+		WaitingFor:   wait.ForLog("Server is ready"),
```

Comment:
suggestion: instead of relying on the log message, we should replace it with a wait for the listening port, which is more deterministic. Thoughts?